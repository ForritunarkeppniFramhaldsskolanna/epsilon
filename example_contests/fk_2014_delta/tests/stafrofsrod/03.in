15
kuxmlasjzc
ujaqcq
pohewtz
m
ldajimflu
ctwtemhhlgh
tvaqgcytxivqq
epjlvq
ckjcnopy
kqvdvx
tofvlaork
yrjbc
qqqsooqkridl
lkxk
hgagfwbxektuo
