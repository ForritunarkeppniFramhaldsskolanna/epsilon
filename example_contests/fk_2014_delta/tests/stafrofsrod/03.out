ckjcnopy
ctwtemhhlgh
epjlvq
hgagfwbxektuo
kqvdvx
kuxmlasjzc
ldajimflu
lkxk
m
pohewtz
qqqsooqkridl
tofvlaork
tvaqgcytxivqq
ujaqcq
yrjbc
